geo.txt

```diff
@@ -56,3 +56,1703 @@ Phys 91(11):957-962
+spectrum of S(CN)2 measured with synchrotron radiation: global analysis of the available high-resolution
+spectroscopic data. J Phys Chem A 117(50):13815-13824
+# Date first added: 2023-07-11 (by Sichao Xiong)
+
+S   C     1.698(6)
+C ≡ N     1.160(9) 
+C   S C   98.6(5)
+S   C ≡ N 175.4(11)
+
+NAME = C2OS3 2010
+# Point Group: Cs
+# Shape: Linear --- Linear
+# Bond lengths: r0     (angstroms)
+# Bond angles:  theta0 (degrees)
+# Source paper: Norooz Oliaee J, Mivehvar F, Dehghany M, Moazzen-Ahmadi N (2010) Geometric isomerism in the OCS-CS2
+complex: Observation of a cross-shaped isomer. J Phys Chem A 114(27):7311-7314
+# Date first added: 2023-07-11 (by Sichao Xiong)
+
+S2C2O  SC1S        3.5553(8)
+C(1)...C(2) = S(2) 104.82(22)
+
+NAME = C2O3
+# Point Group: C2v
+# Shape: Linear --- Linear
+# Bond lengths: r0     (angstroms)
+# Source paper: Sheybani-Deloui S, Barclay AJ, Michaelian KH, McKellar ARW, Moazzen-Ahmadi N (2015) Spectroscopic
+observation of the O-bonded T-shaped isomer of the CO-CO2 dimer and two of its intermolecular frequencies. J
+Chem Phys 143(12):121101/1-121101/4
+[http://dx.doi.org/10.1063/1.4932043]
+# Date first added: 2023-07-11 (by Sichao Xiong)
+
+OCO OC 3.583
+
+NAME = C2O3S 
+# Point Group: Cs
+# Shape: Linear --- Linear
+# Bond lengths: r0     (angstroms)
+# Source paper: a. Dehghany M, Norooz Oliaee J, Afshari M, Moazzen-Ahmadi N, McKellar ARW (2009) Infrared spectra of
+the OCS-CO2 complex: Observation of two distinct slipped near-parallel isomers. J Chem Phys 130(22):224310/1-224310/5 doi: 10.1063/1.3152743 b. Sedo G, van Wijngaarden J (2009) Fourier transform microwave spectra of a “new” isomer of OCS-CO2. J
+Chem Phys 131(4) 044303/1-044303/5
+doi: 10.1063/1.3186756
+# Date first added: 2023-07-11 (by Sichao Xiong)
+
+# O-interior
+
+OCO SCO 3.9862(10)
+θ1      118.0 (44) 
+θ2      135.8 (24)
+
+# MW
+
+OCO SCO 3.9816(16)
+θ1      108.8(46)
+θ2      140.7(46)
+
+# S-interior
+
+OCO SCO 3.5545 (10)
+θ1      69.7 (34)
+θ2      78.7 (25)
+
+# θ1: Angle between Rcm and the O=C=O axis (measured from the inner O end)
+# θ2: Angle between Rcm and the O=C=S axis (measured from the S end).
+
+NAME = C2S4
+# Point Group: D2d
+# Shape: Linear --- Linear
+# Bond lengths: r0     (angstroms)
+# Source paper: Rezaei M, Norooz Oliaee J, Moazzen-Ahmadi N, McKellar ARW (2011) Spectroscopic observation and
+structure of CS2 dimer. J Chem Phys 134(14):144306/1-144306/5
+doi:10.1063/1.3578177
+# Date first added: 2023-07-11 (by Sichao Xiong)
+
+C = C     3.539(7)
+C = C = S 90
+
+NAME = C3Ge2
+# Point Group: Dooh 
+# Shape: Linear
+# Bond lengths: re (angstroms)
+# Source paper: Thorwirth S, Lutter V, Javed AJ, Gauss J, Giesen TF (2016) Gas-phase spectroscopic detection and structural
+elucidation of carbon-rich group 14 binary clusters: Linear GeC3Ge. J Phys Chem A 120(2):254-259
+# Date first added: 2023-07-11 (by Sichao Xiong)
+
+C ≡ Ge 1.7695(1)
+C = C  1.2893
+
```

Comment:
The order of the atoms won't be obvious with just this much information.

Reply:
Not sure with molecules with many duplicate names if it's worth identifying the Cs as C1, C2 and C3. May help with identifying the order but will in some cases unnecessarily clutter the document.